deploy/kubernetes/helm/che/templates/deployment.yaml

```diff
@@ -72,7 +72,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-
+        - name: POD_NAMESPACE
```

Comment:
why not reference it with `metadata.namespace` as done on many places in this file?

Reply:
I've tried that approach first, the result:
```
 ›   Error: Error: Unable to execute helm command helm upgrade --install che --force --namespace che --set global.cheDomain=192.168.39.227.nip.io --set global.tls.useSelfSignedCerts=true 
 ›   --set global.ingressDomain=192.168.39.227.nip.io --set cheImage=quay.io/eclipse/che-server:nightly --set che.disableProbes=false  -f 
 ›   /home/user/.cache/chectl/templates/kubernetes/helm/che/values/tls.yaml /home/user/.cache/chectl/templates/kubernetes/helm/che/ / Error: YAML parse error on 
 ›   che/templates/deployment.yaml: error converting YAML to JSON: yaml: line 52: mapping values are not allowed in this context
 ›   Installation failed, check logs in '/tmp/chectl-logs/1599033986714'
```

Using of `Release.Namespace` is safe as it returns the namespace in which the chart is being deployed.

Reply:
@sparkoo if you know how to overcome that error, I'll use the reference to `metadata.namespace` as it is already used in other places.

Reply:
@mmorhun looks strange. I don't know how to fix it and I'm ok with keeping it as is.